tests/unit/test_bridge_unit.py

```
@@ -0,0 +1,185 @@
+        self.username = username
+        self.password = password
+
+    def subscribe(self, topic, qos=0):
+        self.subscriptions.append((topic, qos))
+
+
+class _FakeCallbackApiVersion:
+    VERSION2 = object()
+
+
+def _load_bridge_module():
```

Ruff (0.15.9)

[warning] Missing return type annotation for private function `_load_bridge_module`

(ANN202)